src/class_loader.cpp

```diff
@@ -58,17 +56,14 @@ std::string systemLibraryPrefix()
 
 std::string systemLibrarySuffix()
 {
-// Poco should be compiled with `#define POCO_NO_SHARED_LIBRARY_DEBUG_SUFFIX`
-// to automatically remove the trailing `d` from the shared library suffix
-//   return Poco::SharedLibrary::suffix();
 #ifdef __linux__
   return ".so";
 #elif __APPLE__
   return ".dylib";
 #elif _WIN32
   return ".dll";
 #else
```

Comment:
It would be better if these could be queried from `rcutils` rather than duplicated here.

Reply:
used rcpputil function https://github.com/ros/class_loader/pull/139/commits/f8d2d730a14f6dfbbfac4ef8c372d0ecd67f67e7